Doc/library/datetime.rst

```diff
@@ -2252,6 +2252,13 @@ Notes:
    day of the week and the ISO year (``%G``) are specified in a
    :meth:`strptime` format string. Also note that ``%G`` and ``%Y`` are not
    interchangeable.
+   
+(9)
```

Comment:
This footnote is not referenced anywhere. I think the behavior of `strptime` when no year, month or day components should be in the main body of the "Behavior of strftime() and strptime()" documentation.